pulsar-client/src/main/java/org/apache/pulsar/client/impl/ReaderBuilderImpl.java

```diff
@@ -229,4 +230,26 @@ private ReaderBuilderImpl(PulsarClientImpl client, ReaderConfigurationData<T> co
         conf.setPoolMessages(poolMessages);
         return this;
     }
+
+    @Override
+    public ReaderBuilder<T> autoUpdatePartitions(boolean autoUpdate) {
+        this.conf.setAutoUpdatePartitions(autoUpdate);
+        return this;
+    }
+
+    @Override
+    public ReaderBuilder<T> autoUpdatePartitionsInterval(int interval, TimeUnit unit) {
+        checkArgument(interval > 0, "interval needs to be > 0");
```

Comment:
Do we need to keep this value at least 1 minute?
If user configs with 1ms, then what happens ?